docs/source/tor.rst

```diff
@@ -62,14 +74,22 @@ And start the system Tor service::
 
     brew services start tor
 
-Open OnionShare. Click the Settings icon. Under "How should OnionShare connect to Tor?" choose "Connect using socket file", and set the socket file to be ``/usr/local/var/run/tor/control.socket``. Under "Tor authentication options" choose "No authentication, or cookie authentication". Click the "Test Settings" button. If all goes well, you should see successfully connected to tor.
+Open OnionShare and click the "⚙" icon in it. Under "How should OnionShare connect to Tor?" choose "Connect using control port", and set
+"Control port" to ``127.0.0.1`` and
+"Port" to ``9051``.
 
-Using a system Tor in Linux
----------------------------
+Under "Tor authentication settings" choose "Password", and set your password.
+(Using 7 words in a sequence like ``comprised stumble rummage work avenging construct volatile`` is a good idea for a password.)
+Click the "Test Connection to Tor" button.
 
-First, install the tor package. If you're using Debian, Ubuntu, or a similar Linux distro, I recommend you use Tor Project's `official repository <https://2019.www.torproject.org/docs/debian.html.en>`_. For example, in Ubuntu 20.04::
+If all goes well, you should see "Connected to the Tor controller".
 
-    sudo su -c "echo 'deb http://deb.torproject.org/torproject.org focal main' > /etc/apt/sources.list.d/torproject.list"
+Using the system's Tor in Linux
+-------------------------------
+
+First, install the Tor package. If you're using Debian, Ubuntu, or a similar Linux distro, It is recommended to use the Tor Project's `official repository <https://2019.www.torproject.org/docs/debian.html.en>`_. For example, in Ubuntu 20.04::
+
+    sudo su -c "echo 'deb https://deb.torproject.org/torproject.org focal main' > /etc/apt/sources.list.d/torproject.list"
```

Comment:
This requires Apt > 1.5, which is in 20.04
https://packages.ubuntu.com/focal/comm/apt
but not in
https://packages.debian.org/jessie/apt or  https://packages.debian.org/stretch/apt
where it requires `apt-transport-https`